docs/notifications.md

```diff
@@ -0,0 +1,96 @@
+# Notifications
+
+### Notification Types
+
+#### Package Releases
+
+Private Packagist will notify you every time we find one or more new releases for one of your packages.
+Notification channels allow you to specify about which private packages you want to get notified and whether you also want to receive notifications for mirrored packages as well.
+Additionally, you can filter for releases by minimum stability: "dev / any" will match any release including commits to a branch whereas "stable" will only match releases considered stable releases by Composer.
```

Comment:
In addition to what? Just remove the word "Additionally"?

Reply:
What releases does Composer consider stable? Maybe add a link to composer docs if you don't want to explain all the options in detail here?

Reply:
Done